packages/sdk/src/cosmwasmclient.ts

```diff
@@ -92,6 +102,39 @@ export interface ContractDetails extends Contract {
   readonly initMsg: object;
 }
 
+/** A transaction that is indexed as part of the transaction history */
+export interface IndexedTx {
+  readonly height: number;
+  readonly hash: string;
+  readonly rawLog: string;
+  readonly logs: readonly Log[];
+  readonly tx: CosmosSdkTx;
+  /** The gas limit as set by the user */
+  readonly gasWanted?: number;
+  /** The gas used by the execution */
+  readonly gasUsed?: number;
+  readonly timestamp: string;
```

Comment:
Please doc what the string format is (I assume same as BlockHeader.time?) There is an issue when parsing in go where you must know if it is supposed to be second, millisecond, microsecond, or nanosecond precision....